airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -64,6 +64,7 @@ class GoogleAdsStream(Stream, ABC):
     def __init__(self, config):
         self.config = config
         self.google_ads_client = GoogleAds(**config)
+        self.conversion_window_days = self.config.get('conversion_window_days')
```

Comment:
we should use `self.config.get('conversion_window_days', self.CONVERSION_WINDOW_DAYS)`. This way application logic (e.g: line 95) doesn't need to perform this every time. 

also small suggestion to rename `CONVERSION_WINDOW_DAYS` to `DEFAULT_CONVERSION_WINDOW_DAYS`